en/docs/observability/observability-overview.md

```diff
@@ -0,0 +1,72 @@
+# Observability Overview
+
+Observability in Choreo provides the capability to visualize and monitor the performance of services deployed on Choreo. Choreo has in-built support to view the overall status, latencies, throughput data,  diagnostic data, and logs. Hence, you can use Choreo Observability to detect and troubleshoot anomalies in your services efficiently. 
+
+## Dashboard
+Choreo observability dashboard gives you more than one way to monitor your services. This section will introduce the dashboards. 
+
+### Dashboard overview
+
+![Dashboard Overview](../assets/img/observability/overview-overall.png){: style="height:55%;width:95%"}
+
+The Observability dashboard allows you to:
+
+1. Observe the throughput and latencies of requests served over a given period.
+2. View the logs generated over a given period.
+3. Observe the flame graph (Diagnostics view) generated over a given period.
+4. View the low-code diagram.
+5.  Trace requests.
+
+!!! note
+    By default, the observability view is private and is only visible to the service owner. If you need to share it, toggle the **Share On** switch, copy the observability URL and share it with the required parties.
+    
+    ![Options Bar](../assets/img/observability/options-bar.png){: style="height:120%;width:90%"}
```

Comment:
Due to some reason this image is not appearing in `View file` mode in PR. Double-check this.

Reply:
Checked on the IDE view mode and the compiled view locally, works fine.
![Screenshot from 2021-06-01 12-05-43](https://user-images.githubusercontent.com/7173347/120277535-cbea1080-c2d1-11eb-81b1-744f5f00cdb5.png)
![Screenshot from 2021-06-01 12-05-53](https://user-images.githubusercontent.com/7173347/120277546-cee50100-c2d1-11eb-8956-d449aee7db71.png)